core/blockchain/blockchain_gateway.py

```diff
@@ -151,7 +162,6 @@ def listen_decentralized_learning(self) -> list:
         callback
         """
         def filter(tx: dict) -> str:
-            logging.info("tx: {}".format(tx))
```

Comment:
I presume you'll add these logging lines back in when you're done testing, right?